test/check-application

```diff
@@ -916,7 +916,7 @@ class TestApplication(testlib.MachineCase):
 
         # Delete intermediate image which is in use
         self.execute(auth, f"podman untag {IMG_INTERMEDIATE}")
-        b.click(intermediate_image_sel + " .pf-v5-c-dropdown__toggle")
+        b.click(intermediate_image_sel + " .pf-v5-c-menu-toggle")
```

Comment:
Maybe we should just use a .... helper function

Reply:
This becomes more and more of a yakshave as we repeat the same stuff in `deleteImage`... This feels follow up territory.